src/structures/GuildMember.js

```diff
@@ -95,7 +95,7 @@ class GuildMember extends Base {
    * @readonly
    */
   get partial() {
-    return !this.joinedTimestamp;
+    return this.joinedTimestamp !== null;
```

Comment:
This will mark all joined members as partial, no?

Reply:
💀 I forgot to push again